commons/tool/src/java/org/sakaiproject/commons/tool/entityprovider/CommonsEntityProvider.java

```diff
@@ -500,4 +517,32 @@ private String getCheckedUser() throws EntityException {
     private String escape(String unescaped) {
         return StringEscapeUtils.escapeJava(unescaped);
     }
+
+    private void sendPriorityEmail(String siteId, String postContent){
+        try{
+            Set<String> siteUserIds = siteService.getSite(siteId).getUsers();
+            Set<User> siteUsers = new HashSet<User>();  //will be list of users to email
```

Comment:
Diamond operator here:

```suggestion
            Set<User> siteUsers = new HashSet<>();  //will be list of users to email
```

Reply:
Changed this  as requested!